gnupg-pkcs11-scd/command.c

```diff
@@ -1022,7 +1022,9 @@ gpg_error_t cmd_pksign (assuan_context_t ctx, char *line)
 		INJECT_SHA256,
 		INJECT_SHA384,
 		INJECT_SHA512
-	} inject = INJECT_NONE;
+	} injectType;
+
+	injectType inject = INJECT_NONE;
```

Comment:
now these changes are not required, right?

Reply:
Oh right :) My bad, fixed.